src/client/app/translations/data.js

```diff
@@ -201,7 +201,8 @@ const localeData = {
 		"insufficient.readings": "Insufficient readings data to process comparison for ",
 		"invalid.email.password": "Invalid email/password combination",
 		"invalid.number": "Please submit a valid number (between 0 and 2.0)",
-		"invalid.token.login.or.logout": "Token has expired. Either log in again, or log out.",
+		"invalid.token.login": "Token has expired. Please log in again.",
+		"invalid.token.login.admin": "Token has expired. Please log in again to view this page.",
```

Comment:
This needs to be added to es and fr sections where it is fine to use the needed text in front as shown in others.